src/types/main.ts

```diff
@@ -39,16 +39,18 @@ export interface MarkerLocation {
   location: Location;
 }
 
-export interface IndoorLocation extends MarkerLocation {
+export interface POI extends MarkerLocation {
   buildingId: BuildingId;
   level: number;
-}
-
-export interface POI extends IndoorLocation {
   description: string;
   category: POICategory;
 }
 
+export interface SearchResult extends MarkerLocation {
+  searchString: string;
+  extraString: string;
```

Comment:
searchString and extraString - Why is the type in the name, we don't do this for any other types of string or non-string?

Reply:
Good point, ill rename them

Reply:
Great thank you!